modules/apps/content-dashboard/content-dashboard-web/src/test/java/com/liferay/content/dashboard/web/internal/item/JournalArticleContentDashboardItemTest.java

```diff
@@ -160,6 +176,181 @@ public void testGetAssetTags() {
 			journalArticleContentDashboardItem.getAssetTags());
 	}
 
+	@Test
+	public void testGetDefaultContentDashboardItemAction() throws Exception {
+		JournalArticle journalArticle = _getJournalArticle();
+
+		Mockito.when(
+			journalArticle.getStatus()
+		).thenReturn(
+			WorkflowConstants.STATUS_APPROVED
+		);
+
+		JournalArticleContentDashboardItem journalArticleContentDashboardItem =
+			new JournalArticleContentDashboardItem(
+				null, null,
+				_getContentDashboardItemActionProviderTracker(
+					_getContentDashboardItemActionProvider(
+						ContentDashboardItemAction.Type.VIEW,
+						"http://localhost:8080/view")),
+				null, null, null, journalArticle, new LanguageImpl(), null,
+				new PortalImpl());
+
+		ContentDashboardItemAction contentDashboardItemAction =
+			journalArticleContentDashboardItem.
+				getDefaultContentDashboardItemAction(
+					_getHttpServletRequest(RandomTestUtil.randomLong()));
+
+		Assert.assertEquals(
+			"http://localhost:8080/view", contentDashboardItemAction.getURL());
+	}
+
+	@Test
+	public void testGetDefaultContentDashboardItemActionWithApprovedAndDraftStatusAndNotOwnerUser()
+		throws Exception {
+
+		JournalArticle journalArticle1 = _getJournalArticle();
+
+		Mockito.when(
+			journalArticle1.getStatus()
+		).thenReturn(
+			WorkflowConstants.STATUS_APPROVED
+		);
+
+		JournalArticle journalArticle2 = _getJournalArticle();
+
+		Mockito.when(
+			journalArticle2.getStatus()
+		).thenReturn(
+			WorkflowConstants.STATUS_DRAFT
+		);
+
+		Mockito.when(
+			journalArticle2.getVersion()
+		).thenReturn(
+			1.1
+		);
+
+		JournalArticleContentDashboardItem journalArticleContentDashboardItem =
+			new JournalArticleContentDashboardItem(
+				null, null,
+				_getContentDashboardItemActionProviderTracker(
+					_getContentDashboardItemActionProvider(
+						ContentDashboardItemAction.Type.VIEW,
+						"http://localhost:8080/view"),
+					_getContentDashboardItemActionProvider(
+						ContentDashboardItemAction.Type.EDIT,
+						"http://localhost:8080/edit")),
+				null, null, null, journalArticle1, new LanguageImpl(),
+				journalArticle2, new PortalImpl());
+
+		ContentDashboardItemAction contentDashboardItemAction =
+			journalArticleContentDashboardItem.
+				getDefaultContentDashboardItemAction(
+					_getHttpServletRequest(RandomTestUtil.randomLong()));
+
+		Assert.assertEquals("viewURL", contentDashboardItemAction.getURL());
```

Comment:
Should it be ""http://localhost:8080/view""?

Reply:
Yeahhh!!!!!